regression/cbmc/pragma_cprover_enable_all/test.desc

```diff
@@ -0,0 +1,27 @@
+CORE broken-smt-backend
```

Comment:
Just curious: any idea why this is failing with the SMT back-end?

Reply:
I honestly have no idea. Seems related to some constructs being rejected by the SMT parser